src/Plugin/Replace/Handler.php

```diff
@@ -57,106 +80,103 @@ public function run(): Task {
 	}
 
 	/**
-	 * @param  Client  $manticoreClient
-	 * @param  string  $table
-	 * @return array <int, array<string, string>>
+	 * @param Client $manticoreClient
+	 * @param string $table
+	 * @return array<int|string, array{type:string, properties:string}>
 	 * @throws ManticoreSearchClientError
 	 */
 	private static function getFields(Client $manticoreClient, string $table): array {
 		$descResult = $manticoreClient
-			->sendRequest('DESC '.$table)
+			->sendRequest('DESC ' . $table)
 			->getResult();
 
 		if (is_array($descResult[0])) {
-			return $descResult[0]['data'];
+			$fields = [];
+			/** @var array{Type:string, Properties:string, Field:string} $field */
+			foreach ($descResult[0]['data'] as $field) {
+				$fields[$field['Field']] = ['type' => $field['Type'], 'properties' => $field['Properties']];
+			}
+
+			return $fields;
 		}
 
-		return [];
+		throw ManticoreSearchClientError::create('Table hasn\'t fields');
```

Comment:
Try to avoid escaping, when we have ' inside, we can use double quotes to make it more clear to read and edit, "Table hasn't fields"